### ISSUE TITLE
[Regression] Missing virutalenv in PATH variable after upgrade from 2021.5.29 to 2021.11.5.post0

### ISSUE THREAD

**Issue by earthday** [pypa/pipenv#4841]
### Issue description

1. Create a shell script
```
# file a.sh
#!/usr/bin/env bash

pipenv --version
echo $PATH
pytest
```

2. To install `pytest`
```
pipenv install pytest
```

3. Run `pipenv run ./a.sh` with pipenv with version `2021.5.29`

```
[root@01f20e1eec40 x]# pipenv run ./a.sh
pipenv, version 2021.5.29
/root/.local/share/virtualenvs/x-fJamIjPy/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
========================================================================= test session starts =========================================================================
platform linux -- Python 3.8.9, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /tmp/x
collected 0 items

======================================================================== no tests ran in 0.01s ========================================================================
```

It works well.

BUT, after upgrading `pipenv` version to `2021.11.5.post0`, it failed

```
[root@01f20e1eec40 x]# pipenv run ./a.sh
pipenv, version 2021.11.5.post0
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
./a.sh: line 5: pytest: command not found
```

So, that means the `virtualenv` path is missing.

### Expected result

This is a regression issue, I hope the virtual env path can be added in `PATH` as the previous version did.

### Actual result

Command failed, and the reason is `virtualenv` path is missing in `PATH`.

### Steps to replicate

See above description.

-------------------------------------------------------------------------------

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2021.11.5.post0'`

Pipenv location: `'/usr/local/lib/python3.8/site-packages/pipenv'`

Python location: `'/usr/bin/python3'`

Python installations found:

  - `3.8.9`: `/usr/local/bin/python3.8`
  - `3.8.9`: `/usr/bin/python3.8`
  - `3.8.9`: `/usr/bin/python3`
  - `3.8.9`: `/bin/python3.8`
  - `3.8.9`: `/bin/python3`
  - `2.7.5`: `/usr/bin/python`
  - `2.7.5`: `/usr/bin/python2.7`
  - `2.7.5`: `/usr/bin/python2`
  - `2.7.5`: `/bin/python`
  - `2.7.5`: `/bin/python2.7`
  - `2.7.5`: `/bin/python2`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.8.9',
 'os_name': 'posix',
 'platform_machine': 'x86_64',
 'platform_python_implementation': 'CPython',
 'platform_release': '5.10.70-1-MANJARO',
 'platform_system': 'Linux',
 'platform_version': '#1 SMP PREEMPT Thu Sep 30 15:29:01 UTC 2021',
 'python_full_version': '3.8.9',
 'python_version': '3.8',
 'sys_platform': 'linux'}
```

System environment variables:

  - `HOSTNAME`
  - `TERM`
  - `LC_ALL`
  - `LS_COLORS`
  - `PYTHON_VERSION`
  - `PATH`
  - `PWD`
  - `LANG`
  - `HOME`
  - `SHLVL`
  - `_`
  - `OLDPWD`
  - `PIP_SHIMS_BASE_MODULE`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `PYTHONDONTWRITEBYTECODE`
  - `PIP_PYTHON_PATH`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`

Pipenv–specific environment variables:

Debug–specific environment variables:

  - `PATH`: `/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG`: `en_US.UTF-8`
  - `PWD`: `/tmp/x`

---------------------------

Contents of `Pipfile` ('/tmp/x/Pipfile'):

```toml
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
pytest = "*"

[dev-packages]

[requires]
python_version = "3.8"

```

Contents of `Pipfile.lock` ('/tmp/x/Pipfile.lock'):

```json
{
    "_meta": {
        "hash": {
            "sha256": "976a2ca0f737f325f408ba96dbdd2082722d52709e42df5c38f7096e58a834da"
        },
        "pipfile-spec": 6,
        "requires": {
            "python_version": "3.8"
        },
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {
        "attrs": {
            "hashes": [
                "sha256:149e90d6d8ac20db7a955ad60cf0e6881a3f20d37096140088356da6c716b0b1",
                "sha256:ef6aaac3ca6cd92904cdd0d83f629a15f18053ec84e6432106f7a4d04ae4f5fb"
            ],
            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'",
            "version": "==21.2.0"
        },
        "iniconfig": {
            "hashes": [
                "sha256:011e24c64b7f47f6ebd835bb12a743f2fbe9a26d4cecaa7f53bc4f35ee9da8b3",
                "sha256:bc3af051d7d14b2ee5ef9969666def0cd1a000e121eaea580d4a313df4b37f32"
            ],
            "version": "==1.1.1"
        },
        "packaging": {
            "hashes": [
                "sha256:096d689d78ca690e4cd8a89568ba06d07ca097e3306a4381635073ca91479966",
                "sha256:14317396d1e8cdb122989b916fa2c7e9ca8e2be9e8060a6eff75b6b7b4d8a7e0"
            ],
            "markers": "python_version >= '3.6'",
            "version": "==21.2"
        },
        "pluggy": {
            "hashes": [
                "sha256:4224373bacce55f955a878bf9cfa763c1e360858e330072059e10bad68531159",
                "sha256:74134bbf457f031a36d68416e1509f34bd5ccc019f0bcc952c7b909d06b37bd3"
            ],
            "markers": "python_version >= '3.6'",
            "version": "==1.0.0"
        },
        "py": {
            "hashes": [
                "sha256:51c75c4126074b472f746a24399ad32f6053d1b34b68d2fa41e558e6f4a98719",
                "sha256:607c53218732647dff4acdfcd50cb62615cedf612e72d1724fb1a0cc6405b378"
            ],
            "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'",
            "version": "==1.11.0"
        },
        "pyparsing": {
            "hashes": [
                "sha256:c203ec8783bf771a155b207279b9bccb8dea02d8f0c9e5f8ead507bc3246ecc1",
                "sha256:ef9d7589ef3c200abe66653d3f1ab1033c3c419ae9b9bdb1240a85b024efc88b"
            ],
            "markers": "python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'",
            "version": "==2.4.7"
        },
        "pytest": {
            "hashes": [
                "sha256:131b36680866a76e6781d13f101efb86cf674ebb9762eb70d3082b6f29889e89",
                "sha256:7310f8d27bc79ced999e760ca304d69f6ba6c6649c0b60fb0e04a4a77cacc134"
            ],
            "index": "pypi",
            "version": "==6.2.5"
        },
        "toml": {
            "hashes": [
                "sha256:806143ae5bfb6a3c6e736a764057db0e6a0e05e338b5630894a5f779cabb4f9b",
                "sha256:b3bda1d108d5dd99f4a20d24d9c348e91c4db7ab1b749200bded2f839ccbe68f"
            ],
            "markers": "python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2'",
            "version": "==0.10.2"
        }
    },
    "develop": {}
}

```
</details>

**Comment by rtoma** [comment]
Observed same regression. As temporary fix workaround, I've pinned pipenv to 2021.5.29

**Comment by frostming** [comment]
Duplicate of #4838
